backends/p4tools/modules/testgen/core/small_step/table_stepper.cpp

```diff
@@ -578,6 +576,15 @@ TableStepper::TableStepper(ExprStepper *stepper, const IR::P4Table *table)
         const auto *action = table->getActionList()->actionList.at(index);
         properties.actionIdMap.emplace(action->controlPlaneName(), index);
     }
+
+    // Set the appropriate properties when the table is immutable, meaning it has constant entries.
+    TableUtils::checkTableImmutability(*table, properties);
+
+    // If the table is in the set of entities to skip, we set it immutable.
+    // P4Testgen will not add a control plane entry for this table.
+    if (TestgenOptions::get().skippedControlPlaneEntities.count(properties.tableName) != 0U) {
```

Comment:
Do standard sets not have a contains?! It looks like it does, but maybe p4c doesn't support C++20 yet? 

If so, would still much prefer find... This is weird.

Reply:
No, this is a C++20 feature. It is common to use `count` for this purpose because it is a bit more readable than find, so purely a force of habit on my side. I have never quite figured out whether it is slower or not, but both ostensibly have logarithmic complexity. Might be better to use find for consistency.

Reply:
That would be my take, but I'll allow whichever you prefer.